std_msgs/Header header
vapaa_cruiser/objectDetect[] object_array